op-node/flags/flags.go

```diff
@@ -19,62 +19,79 @@ import (
 
 const EnvVarPrefix = "OP_NODE"
 
+const (
+	RollupCategory     = "ROLLUP"
+	L1RPCCategory      = "L1 RPC"
+	SequencerCategory  = "SEQUENCER"
+	P2PCategory        = "NETWORKING"
+	OperationsCategory = "LOGGING, METRICS, DEBUGGING, AND API"
+	PlasmaCategory     = "PLASMA (EXPERIMENTAL)"
+)
+
 func prefixEnvVars(name string) []string {
 	return []string{EnvVarPrefix + "_" + name}
 }
 
 var (
 	/* Required Flags */
 	L1NodeAddr = &cli.StringFlag{
-		Name:    "l1",
-		Usage:   "Address of L1 User JSON-RPC endpoint to use (eth namespace required)",
-		Value:   "http://127.0.0.1:8545",
-		EnvVars: prefixEnvVars("L1_ETH_RPC"),
+		Name:     "l1",
+		Usage:    "Address of L1 User JSON-RPC endpoint to use (eth namespace required)",
+		Value:    "http://127.0.0.1:8545",
+		EnvVars:  prefixEnvVars("L1_ETH_RPC"),
+		Category: RollupCategory,
 	}
 	L2EngineAddr = &cli.StringFlag{
-		Name:    "l2",
-		Usage:   "Address of L2 Engine JSON-RPC endpoints to use (engine and eth namespace required)",
-		EnvVars: prefixEnvVars("L2_ENGINE_RPC"),
+		Name:     "l2",
+		Usage:    "Address of L2 Engine JSON-RPC endpoints to use (engine and eth namespace required)",
+		EnvVars:  prefixEnvVars("L2_ENGINE_RPC"),
+		Category: RollupCategory,
 	}
 	L2EngineJWTSecret = &cli.StringFlag{
 		Name:        "l2.jwt-secret",
 		Usage:       "Path to JWT secret key. Keys are 32 bytes, hex encoded in a file. A new key will be generated if the file is empty.",
 		EnvVars:     prefixEnvVars("L2_ENGINE_AUTH"),
 		Value:       "",
 		Destination: new(string),
+		Category:    RollupCategory,
 	}
 	/* Optional Flags */
 	BeaconAddr = &cli.StringFlag{
 		Name:     "l1.beacon",
 		Usage:    "Address of L1 Beacon-node HTTP endpoint to use.",
 		Required: false,
 		EnvVars:  prefixEnvVars("L1_BEACON"),
+		Category: RollupCategory,
 	}
 	BeaconHeader = &cli.StringFlag{
 		Name:     "l1.beacon-header",
 		Usage:    "Optional HTTP header to add to all requests to the L1 Beacon endpoint. Format: 'X-Key: Value'",
 		Required: false,
 		EnvVars:  prefixEnvVars("L1_BEACON_HEADER"),
+		Category: RollupCategory,
```

Comment:
Why are the `l1.beacon*` flags in the rollup category instead of the `L1 RPC` category? We might want to put them both under some `L1 ...` category?

Reply:
`l1.beacon` should go into rollup. The other could maybe move. High level the `rollup` category is everything you need to set for the rollup & `L1 RPC` are optional flags for tuning the client behavior

Reply:
I would still expect all `---l1.*` flags to be under the same category? E.g. `--l1` itself is in rollup, whereas flags tweaking it, like `rpckind` are now under `L1 RPC`.
And if we keep this split, then we should move all optional `--l1.beacon*` flags into the `L1 RPC` cat.